File: StarWarsTcg.Web/src/app/models/CardFrequency.ts
import { Card } from "./Card";

export class CardFrequency {
    cardId: number = 0;
    frequency: number = 0;
    totalQuantity: number = 0;
    card: Card | null = null;
    
    constructor(values: Object = {}) {
        Object.assign(this, values);
    }
}

File: StarWarsTcgApi/StarWarsTcgApi.Domain/Models/CardFrequency.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace StarWarsTcgApi.Domain.Models
{
    public class CardFrequency
    {
        [Required]
        public string CardId { get; set; } = string.Empty;
        public int Frequency { get; set; }
        public int TotalQuantity { get; set; }
    }
}